docs/using.rst

```diff
@@ -807,6 +862,15 @@ Certbot accepts a global configuration file that applies its options to all invo
 of Certbot. Certificate specific configuration choices should be set in the ``.conf``
 files that can be found in ``/etc/letsencrypt/renewal``.
 
+Renewal configuration files store all values specified when a certificate was first
+issued. This includes things like key size, domains, authentication and installation
+method, and pre/post/deploy hooks. It is not recommended to manually edit these values
+as incorrectly set values can cause renewal failure. If you would like to update the
+information set in a renewal configuration file it is recommended that you re-issue
+the certificate with the values that you would prefer. If you do choose to edit the
+configuration file directly it is recommended that you call ``certbot renew --dry-run``
+to ensure that you haven't broken that certificate's ability to renew.
```

Comment:
Do we want this here? A lot of this information is found under "Modifying the Renewal Configuration File". This section heading can be renamed if you want.

Reply:
I still think this section is out of place. Personally what I'd like to see is to have a new sentence added to the previous paragraph saying something like:

"To learn more about setting the values in this file see Modifying the Renewal Configuration File"

and then have this paragraph merged into that section.

Reply:
These comments are unaddressed. Do you disagree with me or did you miss them?